### ISSUE TITLE
[Adaptive Cards - Sample - Inputs with validation] - In high contrast Aquatic theme, 'Calendar' icon present on the 'Due Date' edit field is not visible on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9310]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Pre-requisites:**
Go to system Settings-> Accessibility-> Contrast themes-> Select Aquatic/Desert theme

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'Inputs with validation' control present on the left navigation using tab key and activate using enter key.
4. Navigate to the 'Calendar' icon present on the 'Due Date' edit field using tab key.
5. Verify that, in high contrast Aquatic theme, 'Calendar' icon present on the 'Due Date' edit field is visible on the page or not.

**Actual Result:**
In high contrast Aquatic theme, 'Calendar' icon present on the 'Due Date' edit field is not visible on the page.

**Note:**
Same issue replicates for all the 'Calendar' icon present throughout the Adaptive Cards page.

**Expected Result:**
In high contrast Aquatic theme, 'Calendar' icon present on the 'Due Date' edit field should be properly visible on the page.

**User Experience:**
Color blind users who rely on high contrast themes to access the web content will be impacted to identify the control if in high contrast Aquatic theme, 'Calendar' icon present on the 'Due Date' edit field is not visible on the page. Due to which users will face difficulty to identify the control and might miss the information and functionality of controls present on the page.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/42ac0498-d0df-406e-95a2-e7582c8329e5)

### Card JSON

```JSON
NA
```
